charts/perses/templates/tests/test.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.testFramework.enabled -}}
+apiVersion: v1
+kind: Pod
+metadata: 
+  name: {{ .Release.Name }}-test
+  namespace: {{ .Release.Namespace }}
+  labels: 
+    type: helm-chart-test
+  annotations:
+    "helm.sh/hook": test
```

Comment:
Again a question. Some people can use helm for the templating only, without having helm installed in the platform. 
For them I guess these hooks won't work, right ?

Reply:
You're correct! When using Helm templates solely for Kubernetes manifest generation, the `helm.sh/hook: test annotations` won't have any effect. These hooks are specifically processed by the Helm binary during helm test execution.
However, since this is a Perses helm-charts repository, we expect users to be using these charts with Helm to manage Perses application(s) in their Kubernetes clusters, rather than just using them as templates.